docs/arch/adr-008-weave-to-flannel-migration.md

```diff
@@ -37,3 +39,6 @@ Proposed
 ## Consequences
 
 Users will be able to move from Weave to Flannel, but downtime will need to be scheduled to do so.
+
+If a migration failed, manual remediation would be required.
```

Comment:
can you go into more detail what this means? i assume the cluster is down.

Reply:
Yeah cluster would be very down
'why down' depends on where things broke, though